app/scripts/services/globalPromotionService.ts

```diff
@@ -85,21 +86,43 @@ export class GlobalPromotionService {
     this.globalPromotionsCache.put(cacheKey, [...cachedData, newPromotion]);
   }
 
+  private hasPromotionsAvailable(
+    conference: Conference,
+    hasEligibleRegistrants: boolean,
```

Comment:
This is subjective, but I would remove the `hasEligibleRegistrants` argument here and change `hasPromotionsForConference` to `return hasEligibleRegistrants && this.hasPromotionsAvailable(conference);`. To me at least it lets `hasPromotionsAvailable` just handle determining whether a conference has any matching ministry promotions. And it's not any more verbose.